docs/framework/index.yml

```diff
@@ -0,0 +1,132 @@
+          - text: Older versions of .NET Framework
+            url: /previous-versions/dotnet/
+      - linkListType: concept
+        links:
+          - text: Deployment guide for developers
+            url: deployment/deployment-guide-for-developers.md
+
+  # Card
+  - title: Install .NET Framework
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Installation guide 
+            url: install/index.md
+      - linkListType: how-to-guide
+        links:
+          - text: Install .NET Framework on Windows 10 
+            url: install/on-windows-10.md
+          - text: Install .NET Framework 3.5
+            url: install/dotnet-35-windows-10.md
+          - text: Install the developer pack or redistributable 
+            url: install/guide-for-developers.md
+          - text: Troubleshoot blocked installations
+            url: install/troubleshoot-blocked-installations-and-uninstallations.md
+
+  # Card
+  - title: Create Windows service apps
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Introduction to Windows service apps
+            url: windows-services/introduction-to-windows-service-applications.md
+      - linkListType: tutorial
+        links:
+          - text: Create a Windows service app 
+            url: windows-services/walkthrough-creating-a-windows-service-application-in-the-component-designer.md
+      - linkListType: how-to-guide
+        links:
+          - text: Install and uninstall Windows services 
+            url: windows-services/how-to-install-and-uninstall-services.md
+
+  # Card
+  - title: Create WPF apps
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Overview of WPF
+            url: wpf/index.md
+      - linkListType: tutorial
+        links:
+          - text: Create your first WPF app in Visual Studio
+            url: wpf/getting-started/walkthrough-my-first-wpf-desktop-application.md
+
+  # Card
+  - title: Create Windows Forms apps
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Overview of Windows Forms
+            url: winforms/index.md
+      - linkListType: how-to-guide
+        links:
+          - text: Add and remove items from ComboBox, ListBox, and CheckedListBox controls 
+            url: winforms/controls/add-and-remove-items-from-a-wf-combobox.md
+          - text: Make thread-safe calls to controls
+            url: winforms/controls/how-to-make-thread-safe-calls-to-windows-forms-controls.md
+      - linkListType: reference
+        links:
+          - text: Design-time errors in the Windows Forms Designer 
+            url: winforms/controls/design-time-errors-in-the-windows-forms-designer.md
+
+  # Card
+  - title: Work with data using ADO.NET
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Overview of ADO.NET
+            url: data/adonet/ado-net-overview.md
+      - linkListType: concept
```

Comment:
The links in this section seem more like how-to docs than conceptual docs, except perhaps the connection pooling doc